client/src/modules/AutoTest/components/Exercise/Exercise.tsx

```diff
@@ -25,6 +27,24 @@ function responsiveColumns(type: CourseTaskDetailedDtoTypeEnum): ColProps | unde
 
 function Exercise({ githubId, courseId, courseTask, finishTask }: ExerciseProps) {
   const { form, loading, submit, change } = useCourseTaskSubmit(courseId, courseTask, finishTask);
+  const [validationError, setValidationError] = useState(false);
+
+  const values = Form.useWatch([], form);
+
+  useEffect(() => {
+    if (!values || !Object.values(values).every(Boolean)) {
+      return;
+    }
```

Comment:
@ThorsAngerVaNeT 
Do we really need these lines? What will happen if we'll remove them?

Reply:
ah, got it, nvm